src/content/es/2020/http2.md

```diff
@@ -0,0 +1,690 @@
+
+¿Qué pasa con el desglose de la página en sí? Por lo general, hablamos de la diferencia entre contenido propio y de terceros. Tercero se define como contenido que no está bajo el control directo del propietario del sitio, que proporciona funciones como publicidad, marketing o análisis. La definición de terceros conocidos se toma del repositorio <a href="https://github.com/patrickhulce/third-party-web/blob/8afa2d8cadddec8f0db39e7d715c07e85fb0f8ec/data/entities.json5" hreflang="en">third party web</a>.
+
+La Figura 22.7 ordena cada sitio web por la fracción de solicitudes HTTP/2 para terceros conocidos o solicitudes de contenido propio en comparación con otras solicitudes. Existe una diferencia notable, ya que más del 40% de todos los sitios no tienen del todo solicitudes HTTP/2 o gQUIC. Por el contrario, incluso el más bajo 5% de las páginas tiene un 30% de contenido de terceros servido a través de HTTP/2. Esto indica que una gran parte de la amplia adopción de HTTP/2 es impulsada por terceros.
+
+{{ figure_markup(
+  image="http2-first-and-third-party-http2-usage.png",
+  caption="Distribución de la fracción de solicitudes HTTP/2 de terceros y de contenido propio por página.",
+  description="Un gráfico de líneas que compara la fracción de solicitudes HTTP/2 de contenido propio con solicitudes HTTP/2 o gQUIC de terceros. El gráfico ordena los sitios web por fracción de solicitudes HTTP/2. El 45% de los sitios web no tienen solicitudes HTTP/2 de contenido propio. Más de la mitad de los sitios web atienden solicitudes de terceros solo a través de HTTP/2 o gQUIC. El 80% de los sitios web tienen 76% o más solicitudes HTTP/2 o gQUIC de terceros.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1409316276&format=interactive",
+  sql_file="http2_1st_party_vs_3rd_party.sql",
+  sheets_gid="733872185"
+)
+}}
+¿Hay alguna diferencia ente cuáles tipos de contenido se sirven a través de HTTP/2 o gQUIC? La Figura 22.8 muestra, por ejemplo, que el 90% de los sitios web sirven el 100% de fuentes y audio de terceros a través de HTTP/2 o gQUIC, solo el 5% a través de HTTP/1.1 y un 5% son una combinación. La mayoría de los activos de terceros son scripts o imágenes, y se sirven únicamente a través de HTTP/2 o gQUIC en el 60% y el 70% de los sitios web, respectivamente.
+
+{{ figure_markup(
+  image="http2-third-party-http2-usage-by-content-type.png",
+  caption="Fracción de solicitudes HTTP/2 o gQUIC de terceros conocidos por <i lang="en">content-type</i> por sitio web.",
+  description="Un gráfico de barras que compara la fracción de solicitudes HTTP/2 de terceros por <i lang="en">content-type</i>. Todas las solicitudes de terceros se envían a través de HTTP/2 o gQUIC para el 90% de audio y fuentes, 80% de css y video, 70% de html, imágenes y texto y 60% de scripts.",
```

Comment:
When using `lang="en"` it contains quotes so need to switch to single quotes for that:

```suggestion
  caption='Fracción de solicitudes HTTP/2 o gQUIC de terceros conocidos por <i lang="en">content-type</i> por sitio web.',
  description='Un gráfico de barras que compara la fracción de solicitudes HTTP/2 de terceros por <i lang="en">content-type</i>. Todas las solicitudes de terceros se envían a través de HTTP/2 o gQUIC para el 90% de audio y fuentes, 80% de css y video, 70% de html, imágenes y texto y 60% de scripts.',
```